apps/frontend/src/libs/components/search/libs/constants/search-parameter-key.constant.ts

```diff
@@ -0,0 +1,3 @@
+const SEARCH_PARAMETER_KEY = "search";
```

Comment:
we alread have QueryParameterName enum and it should be in it, also let's use q keyword for this, it's more common for search. Like below

```typescript
const QueryParameterName = {
	SEARCH: "q",
} as const;
```